packages/amplify-cli/src/__tests__/extensions/amplify-helpers/execute-provider-utils.test.ts

```diff
@@ -11,7 +12,7 @@ jest.mock('../../../../__mocks__/faked-plugin', () => ({
   },
 }));
 describe('executeProviderUtils', () => {
-  const mockContext = {};
+  const mockContext: $TSAny = {};
```

Comment:
please revert these changes, it will conflict with https://github.com/aws-amplify/amplify-cli/pull/7153

Reply:
done, thanks for catching this